rllib/examples/rl_modules/classes/action_masking_rlm.py

```diff
@@ -1,121 +1,133 @@
 import gymnasium as gym
+from typing import Dict, Optional, Tuple
 
-from ray.rllib.algorithms.ppo.tf.ppo_tf_rl_module import PPOTfRLModule
 from ray.rllib.algorithms.ppo.torch.ppo_torch_rl_module import PPOTorchRLModule
+from ray.rllib.core.columns import Columns
 from ray.rllib.core.rl_module.rl_module import RLModule, RLModuleConfig
-from ray.rllib.policy.sample_batch import SampleBatch
-from ray.rllib.utils.framework import try_import_torch, try_import_tf
+from ray.rllib.utils.framework import try_import_torch
 from ray.rllib.utils.torch_utils import FLOAT_MIN
+from ray.rllib.utils.typing import TensorType
 
 torch, nn = try_import_torch()
-_, tf, _ = try_import_tf()
 
 
-class ActionMaskRLMBase(RLModule):
+class ActionMaskRLM(RLModule):
```

Comment:
super nit: Rename to `ActionMaskingRLModule` (same as filename).

Reply:
Good catch. I like it clean :)